backends/arm/operators/op_addmm.py

```diff
@@ -67,12 +70,7 @@ def define_node(
         input_zp = 0
         if is_quant_node:
             input_node = node.all_input_nodes[1]
-            # rank > 2 linear layer
-            if input_node.target == exir_ops.edge.aten.view_copy.default:
-                quant_node = input_node.all_input_nodes[0]
-            else:
-                quant_node = input_node
-            input_zp = get_quant_node_args(quant_node).zp
+            input_zp = search_quant_arg_upstream(input_node).zp
```

Comment:
Nit: though we are checking this for a quant case, but this function can return None. I feel at least we should have some logging in this upstream / downstream search function for easier debugging?

Reply:
My reasoning about (the lack of) error handling here was to match the earlier behavior with the hope to solve some additional cases. Since we have checked is_quant_node = True, we have assumed before that the node is embedded with dq/ q nodes as it should. It's possible that there is some case where we don't have this for some weird reason but that would have failed earlier as well.